Assets/Scripts/PlatformOneTouch.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class PlatformOneTouch : MonoBehaviour
+{
+    short rnd;
+    private void Start()
+    {
+        rnd = (short)Random.Range(0, 10);
+    }
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        Thread.Sleep(500);
```

Comment:
`Thread.Sleep` je zlý, tým blokuješ celé vlákno, čo v našom prípade by kľudne mohlo zamraziť celú hru. Ak chceš len jednoduché čakanie, je lepšie použiť `WaitForSeconds` v `Coroutine` a tú len potom zavolať.[https://docs.unity3d.com/ScriptReference/WaitForSeconds.html](url)

Reply:
Ano, už som to zmenil lebo pto prensne mrazilo celu hru...